frame/staking/src/lib.rs

```diff
@@ -2480,19 +2517,17 @@ impl<T: Trait> Module<T> {
 		} = ledger;
 
 		if *active_ring != ring_staking_lock.staking_amount {
+			let origin_active_ring = ring_staking_lock.staking_amount;
+
 			ring_staking_lock.staking_amount = *active_ring;
```

Comment:
The lock in staking is tracking the `active_ring`, so it is also working as original active ring before being updated.